Cython/Compiler/Optimize.py

```diff
@@ -2452,7 +2452,11 @@ def _handle_simple_function_str(self, node, function, pos_args):
         """
         if len(pos_args) != 1:
             if len(pos_args) == 0:
-                return ExprNodes.StringNode(node.pos, value=EncodedString(), constant_result='')
+                if node.type is Builtin.unicode_type:
+                    node_type = ExprNodes.UnicodeNode
+                else:
+                    node_type = ExprNodes.BytesNode
```

Comment:
Shouldn't this still be `StringNode` in 3.0.x?

Reply:
Yes it should... Just spotted that myself